mix.exs

```diff
@@ -43,6 +43,7 @@ defmodule Guardian.Mixfile do
   defp deps do
     [{:jose, "~> 1.6"},
      {:poison, ">= 1.3.0"},
+     {:phoenix, "~> 1.2.0-rc.1"},
```

Comment:
@doomspork Right now I don't have access to the machine where I did the changes, but, if I recall well `mix compile` was failing because there is a dependency in `guardian/phoenix/socket.ex` that requires `Phoenix.Socket.assign/3`. wdyt?